creek.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 2.13.0'
   spec.add_development_dependency 'pry'
 
-  spec.add_dependency 'nokogiri', '~> 1.6.0'
+  spec.add_dependency 'nokogiri', '~> 1.7'
```

Comment:
It would be better if this was `>= 1.6` to continue supporting rails 4.2.

Rails 4.2 depends on `rails-dom-testing` which depends on `nokogiri (~> 1.6)`